egs/fisher_swbd/s5/local/chain/run_blstm_6h.sh

```diff
@@ -0,0 +1,190 @@
+#!/bin/bash
+
+# based on run_tdnn_6h.sh
+
+set -e
+
+# configs for 'chain'
+stage=12
+train_stage=-10
+get_egs_stage=-10
+dir=exp/chain/blstm_6h  # Note: _sp will get added to this if $speed_perturb == true.
+decode_iter=
+decode_dir_affix=
+
+# training options
+leftmost_questions_truncate=-1
+chunk_width=150
+chunk_left_context=40
+chunk_right_context=40
+xent_regularize=0.025
```

Comment:
OK

在 2016/4/14 0:48, Vijayaditya Peddinti 写道:
>
> In egs/fisher_swbd/s5/local/chain/run_blstm_6h.sh 
> <https://github.com/kaldi-asr/kaldi/pull/657#discussion_r59583185>:
>
> > +set -e
> > +
> > +# configs for 'chain'
> > +stage=12
> > +train_stage=-10
> > +get_egs_stage=-10
> > +dir=exp/chain/blstm_6h  # Note: _sp will get added to this if $speed_perturb == true.
> > +decode_iter=
> > +decode_dir_affix=
> > +
> > +# training options
> > +leftmost_questions_truncate=-1
> > +chunk_width=150
> > +chunk_left_context=40
> > +chunk_right_context=40
> > +xent_regularize=0.025
>
> could you increase the xent_regularize to 0.1. The results might not 
> vary much so no need to rerun the experiment.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub 
> <https://github.com/kaldi-asr/kaldi/pull/657/files/fc4140efa4727c4ad11cf689ae3faa26a9d0ffa0#r59583185>
>